SPECS/mysql/mysql.spec

```diff
@@ -1,19 +1,18 @@
 Summary:        MySQL.
 Name:           mysql
-Version:        8.0.21
+Version:        8.0.22
 Release:        1%{?dist}
-License:        GPLv2 with exceptions and LGPLv2 and BSD
-Group:          Applications/Databases
+License:        GPLv2 WITH exceptions AND LGPLv2 AND BSD
```

Comment:
>WITH [](start = 22, length = 4)

Nit: I don't think "with" needs to be all-caps here. I'm guessing the linter mistook that for a `[License1] WITH [License2]` kind of thing.

Reply:
Can confirm that this is an issue with the linter- the original License field is correct, I'll make a corrective fix to the linter shortly. Feel free to rollback the change to this line and merge without linting passing.

Reply:
fixed!